Makefile

```diff
@@ -36,16 +38,25 @@ rm-sequential:
 start-server: $(WS_PAR)/mets.xml
 	ocrd workspace -d $(WS_PAR)  -U $(SOCK) server start
 
-rm-parallel:
+stop-server: $(WS_PAR)/mets.xml
+	-ocrd workspace -d $(WS_PAR)  -U $(SOCK) server stop
+
+rm-parallel: stop-server
 	rm -rf $(WS_PAR)
 
-$(WS_PAR)/mets.xml:
+$(WS_PAR)/mets.xml: $(WS)
```

Comment:
We should probably also use `$(WS)/mets.xml` instead of just the directory to avoid weird make behavior for directory targets.

Reply:
Granted, but then I would make a single commit which consistently replaces the `$(WS)` target by `$(WS)/mets.xml`.

Reply:
:+1:

Reply:
done